portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,17 +36,32 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final int REQUESTED_COMMENTS_LIMIT = 1000;
```

Comment:
If you actually had 1000 comments, I think this would be excessive to display. Maybe 100 is better.